sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -132,9 +132,11 @@ object AnalysisContext {
  */
 class Analyzer(
     override val catalogManager: CatalogManager,
-    conf: SQLConf)
+    deprecatedConf: SQLConf)
```

Comment:
Is this change related to what this PR proposes?

Reply:
This change is related to another sub-task of [SPARK-33138](https://issues.apache.org/jira/browse/SPARK-33138) which makes internal classes of SparkSession always using active SQLConf. I opened a seperated PR (https://github.com/apache/spark/pull/30299) and removed this change here.